src/helpers/xml-builder.js

```diff
@@ -277,6 +277,24 @@ const fixupLineHeight = (lineHeight, fontSize) => {
 
 // eslint-disable-next-line consistent-return
 const fixupFontSize = (fontSizeString, docxDocumentInstance) => {
+  if (fontSizeString === 'xx-small') {
```

Comment:
write comment explaining how you came to the mappings for this and what they represent. Link to external documentation if its provided

Reply:
@nicolasiscoding Updated PR